malayalam/crawler-asianet.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+import urllib
+import os
+import re
+from tqdm import tqdm
+from bs4 import BeautifulSoup as bs
+import requests
+import sys
+import time
+from pprint import pprint, pformat
+
+sys.path.append('../')
+
+import config
+from crawler import mkdir, verbose
+from crawler import Crawler, MultiThreadedCrawler2
+from crawler import HTTP, HTTPS
+from aliases import GregorianMonthInEnglishShort as MonthAlias
+
+import logging
+logging.basicConfig(format=config.CONFIG.FORMAT_STRING)
+log = logging.getLogger(__name__)
+log.setLevel(config.CONFIG.LOGLEVEL)
+
+
+uid_ = 0
+name = '{}'.format(uid_)
+    
+class OneIndiaCrawler(MultiThreadedCrawler2):
+
+    def __init__(self, root_url, num_threads=1):
+        root_dir= os.path.abspath(__file__)
+        root_dir = '/'.join(root_dir.split('/')[-2:])
```

Comment:
Could you replace the `/` with `os.sep`.

Otherwise it will end up casuing issues in windows as windows uses `\` for path seperator.